test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -426,6 +426,26 @@ class CharacterTest {
     assertEquals(0x10ffff, Character.toLowerCase(0x10ffff)) // 􏿿 => 􏿿
   }
 
+  @Test def toLowerCase_CodePoint_special_cases(): Unit = {
+    assertEquals(0x0069, Character.toLowerCase(0x0130))
+  }
```

Comment:
The two test methods are intentional, because they test logically different things. In the case of `toLowerCase()`, they happen to test the same set of code points. I have added comments to explain the situation.